backend/api/Areas/Property/Mapping/Parcel/ParcelParcelMap.cs

```diff
@@ -53,6 +48,7 @@ public void Register(TypeAdapterConfig config)
             config.NewConfig<Model.SubdivisionParcelModel, Entity.ParcelParcel>()
                 .EnableNonPublicMembers(true)
                 .Map(dest => dest.ParcelId, src => src.Id)
+                .Map(dest => dest.Parcel.PID, src => ParcelConverter.ConvertPID(src.PID))
```

Comment:
We need to pass through the parent parcel pid so that it can be used in place of whatever pid is currently associated to the subdivision.